app/javascript/stylesheets/buildings.scss

```diff
@@ -109,11 +105,13 @@
 }
 
 
-.questionaire-category-container {
+.questionnaire-category-container {
```

Comment:
YOU ARE A CSS GODDESS 🙏

Reply:
SO MUCH STYLING IN ONE PR

Reply:
I love questionaires and portfoios that are styled by Divi 😩